### PR TITLE
Jb initial commit

### PR DESCRIPTION
**Pull request by jbutcher21**
initial commit and documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,149 @@
-# g2explorer
\ No newline at end of file
+# g2explorer
+
+## Overview
+
+The [G2Explorer.py](G2Explorer.py) utility is an interactive command line utility that allows a user to ...
```
Comment (antaenc) [inline README.md:5]: Remove 3 dots, colon instead

File: README.md [hunk 2]
```diff
@@ -1 +1,149 @@
...
+- **search** - Find an entity by name, DOB, address, etc.
```
Comment (antaenc) [inline README.md:6]: Spell out date of birth

File: README.md [hunk 3]
```diff
@@ -1 +1,149 @@
...
+- **get** - Display an entity's records in either summary or detail format.
+- **compare** – Compare two or more entities side by side.
+- **why** – Display the internal engine keys and stats about an entity to help determine why it did or did not match another. 
+- **export** – Export the json records that make up an entity for debugging or reingesting after tweaks are made.
```
Comment (antaenc) [inline README.md:10]: configuration changes instead of tweaks

File: README.md [hunk 4]
```diff
@@ -1 +1,149 @@
...
+
+It can also be used to view the reports generated by the following companion projects ...
```
Comment (antaenc) [inline README.md:12]: 3 dots, use colon

File: README.md [hunk 5]
```diff
@@ -1 +1,149 @@
...
+
+## Contents
+
+1. [Prerequisites](#Prerequisites)
+2. [Installation](#Installation)
+3. [Typical use](#Typical-use)
+4. [Exploring data](#Exploring-data)
+
+### Prerequisites
+- python 3.6 or higher
```
Comment (antaenc) [inline README.md:46]: Uppercase Python

File: README.md [hunk 6]
```diff
@@ -1 +1,149 @@
...
+- Senzing API version 2.4 or higher
+- python pretty table module (pip3 install ptable)
+
+### Installation
+
+1. Simply place the the following files in a directory of your choice ...
```
Comment (antaenc) [inline README.md:52]: Double the

File: README.md [hunk 7]
```diff
@@ -1 +1,149 @@
...
+- The -s snapshot_file_name parameter is for convenience if you just took a snapshot and want to immediately view it.
+- The -a audit_file_name parameter is for convenience if you just ran an audit and want to immediately view it.
+- The -d debug_output parameter turns on debug to either screen or to a file name.  This shows you what the API is returning 
+as you search and get records.  
+-- use -d SCREEN to print the debug output to the screen
+-- or -d /path/to/filename.txt to print the debug output to whatever file you choose.
+
+### Exploring data
+
+The best documentation for how to use the G2Explorer and its companion projects is located here ...
```
Comment (antaenc) [inline README.md:78]: Further documentation for using G2Explorer and it's companion utilities can be located at:

File: README.md [hunk 8]
```diff
@@ -1 +1,149 @@
...
+- https://senzing.zendesk.com/hc/en-us/sections/360009388534-Exploratory-Data-Analysis-EDA- 
+
+But here are some basics ..
```
Comment (antaenc) [inline README.md:81]: Getting started with some basic uses:

File: README.md [hunk 9]
```diff
@@ -1 +1,149 @@
...
+    search {"name_org": "ABC Company"}
+    search {"name_last": "Smith", "name_first": "Joe", "date_of_birth": "1992-12-10"}
+    search {"name_org": "ABC Company", "addr_full": "111 First St, Anytown, USA 11111"}
+
+Notes: 
+    Searching by name alone may not locate a specific entity.
+    Try adding a date of birth, address, or phone number if not found by name alone.
+```
+
+Next thing to do is to just start exploring your database!  Here are a few tips ...
```
Comment (antaenc) [inline README.md:127]: The next thing to do is start exploring

3 dots

**Review by antaenc — CHANGES_REQUESTED** [review]
Some small cleanup

**Comment by jbutcher21** [comment]
Changes made, Ant, please review and approve.

**Review by antaenc — APPROVED** [review]

**Comment by antaenc** [comment]
Changes approved

**Pull request merged**